CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+# 2.13.0
```

Comment:
don't forget to add the date as we want this to be released:
`# 2.13.0 (2020-11-13)`